packages/react/src/components/List/HierarchyList/HierarchyListReorderModal/HierarchyListReorderModal.jsx

```diff
@@ -16,10 +16,6 @@ const propTypes = {
   selectedIds: PropTypes.arrayOf(PropTypes.string).isRequired,
   /** Internationalization text */
   i18n: PropTypes.shape({
-    /** String e.g. '%d items selected' that gets %d replaced by selected count or
-     * function receiving the selectedCount as param:
-     * (selectedCount) => `${selectedCount} items selected` */
-    itemsSelected: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
     itemSelected: PropTypes.string,
```

Comment:
You're right. There are a few others that aren't being used either. I'll remove those. I'm not sure I follow how this would cause any issues downstream though. The function isn't being called anywhere in `HierarchyListReorderModal` and `HierarchyListReorderModal` isn't being exported. Where's the potential issue?

Reply:
Sorry, I thought they were exported. Please ignore my downstream  comment above.